packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -284,68 +288,94 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
     }
   }
 
-  /// Handles a key pressed `event` in the given `context`.
+  /// Returns the [Intent], if any, that matches the current set of pressed
+  /// keys.
   ///
-  /// The optional `keysPressed` argument provides an override to keys that the
-  /// [RawKeyboard] reports. If not specified, uses [RawKeyboard.keysPressed]
-  /// instead.
+  /// Returns null if no intent matches the current set of pressed keys.
   ///
-  /// If a key mapping is found, then the associated action will be invoked
-  /// using the [Intent] that the [LogicalKeySet] maps to, and the currently
-  /// focused widget's context (from [FocusManager.primaryFocus]).
-  ///
-  /// The object returned is the result of [Action.invoke] being called on the
-  /// [Action] bound to the [Intent] that the key press maps to, or null, if the
-  /// key press didn't match any intent.
-  @protected
-  bool handleKeypress(
-    BuildContext context,
-    RawKeyEvent event, {
+  /// Defaults to a set derived from [RawKeyboard.keysPressed] if `keysPressed` is
+  /// not supplied.
+  Intent? find({
```

Comment:
Wondering: Should this still be public API? Seems like we no longer need to expose it?

Reply:
Hmm. Yes, I suppose I could make it private. It does seem generally useful, but I suppose we shouldn't complicate the public API until we have a need to.